packages/ignite/src/commands/doctor.js

```diff
@@ -1,49 +1,82 @@
 // @cliDescription Checks your dev environment for dependencies.
 // ----------------------------------------------------------------------------
 
-const R = require('ramda')
-const Shell = require('shelljs')
+const { split, last, replace, head, match } = require('ramda')
+const header = require('../brand/header')
+const os = require('os')
 
 module.exports = async function (context) {
-  const { filesystem: { exists, read } } = context
-  const { debug } = context.print
+  // fistful of features
+  const {
+    filesystem: { read },
+    system: { run, which },
+    print: { colors, info, table },
+    strings: { padEnd }
+  } = context
 
   // ... be the CLI you wish to see in the world
   const platform = process.platform
-  const ignitePath = R.trim(Shell.which('ignite') || '')
-  const igniteVersion = R.trim(Shell.exec('ignite version', { silent: true }).stdout)
-  const nodePath = R.trim(Shell.which('node'))
-  const npmVersion = R.trim(Shell.exec('npm --version', { silent: true }).stdout)
-  const npmPath = R.trim(Shell.which('npm'))
-  const nodeVersion = R.trim(Shell.exec('node --version', { silent: true }).stdout)
-  const ggPackageFile = `${process.cwd()}/node_modules/gluegun/package.json`
-  const ggVersion = exists(ggPackageFile) ? read(ggPackageFile, 'json').version : '¯\\_(ツ)_/¯'
-
-  const rnCli = R.split(/\s/, R.trim(Shell.exec('react-native --version', { silent: true }).stdout))[1] // lulz
-
-  const rnPackageFile = `${process.cwd()}/node_modules/react-native/package.json`
-  const appReactNativeVersion = Shell.test('-f', rnPackageFile) ? require(rnPackageFile).version : '¯\\_(ツ)_/¯'
-
-  const body = `
-\`\`\`
-Computer
-  Platform: ${platform}
-Ignite
-  Version: ${igniteVersion}
-  Path: ${ignitePath}
-Node
-  Version: ${nodeVersion}
-  Path: ${nodePath}
-NPM
-  Version: ${npmVersion}
-  Path: ${npmPath}
-Gluegun
-  Version: ${ggVersion}
-React Native CLI
-  Version: ${rnCli}
-App
-  React Native Version: ${appReactNativeVersion}
-\`\`\`
-`
-  debug(body, 'Ignite Info')
+  const arch = os.arch()
+  const cpus = os.cpus() || []
+  const firstCpu = head(cpus) || {}
+  const cpu = `${firstCpu.model}`
+  const cores = `${cpus.length} cores`
+  const ignitePath = which('ignite')
+  const igniteVersion = await run('ignite version', { trim: true })
+  const nodePath = which('node')
+  const npmVersion = await run('npm --version', { trim: true })
+  const npmPath = which('npm')
```

Comment:
npm comes with node since version 4.  I'd be surprised to see someone without a valid npm, but we could check for that too if you want.

Ya special permissions aren't required anymore because we're not writing to any system directory.